Library/Homebrew/test/test_integration_cmds.rb

```diff
@@ -689,4 +689,31 @@ def test_unlink
     cmd("install", "testball")
     assert_match "Would remove", cmd("unlink", "--dry-run", "testball")
   end
+
+  def test_pull
+    assert_match "You meant `git pull --rebase`.", cmd_fail("pull", "--rebase")
+    assert_match "This command requires at least one argument", cmd_fail("pull")
+    assert_match "Not a GitHub pull request or commit",
+      cmd_fail("pull", "0")
```

Comment:
Cool. I split the tests and added `skip if ENV["HOMEBREW_NO_GITHUB_API"]` to `test_pull`. Does that work? Also, should a `skip` be added to `test_search`?